modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -167,7 +167,7 @@ export async function createUserSignatureShare(
   oShare: OShare,
   dShare: DShare,
   message: Buffer,
-  hash: Hash = createKeccakHash('keccak256')
+  hash: Hash = createKeccakHash('keccak256') as unknown as Hash
```

Comment:
`createKeccakHash` should be implementing the `Hash` interface, Causing some issues?

Reply:
Yeah, there's insufficient overlap, it's missing the property "copy" or something? Not sure if this is just a types issue, but tests still pass so I think this should work. I can revisit it when I try and add 18 support, or if anyone else wants to look into this that would be great too.